### PR TITLE
exchange currency layout

### PR DESCRIPTION
**Pull request by jonnyalexbh**
## Summary
- exchange currency layout

## Trello Card
N/A

## Screenshot
Screen Shot 2019-12-11 at 6 39 04 PM

### CODE REVIEW

File: plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.scss [hunk 1]
```diff
@@ -0,0 +1,107 @@
+
+
+
+@import '../../../scss/variables/colors';
+@import '../../../scss/variables/fonts';
```
Comment (betoobernal) [inline plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.scss]: use the absolute route.

File: plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.scss [hunk 2]
```diff
@@ -0,0 +1,107 @@
+
+
+
```
Comment (betoobernal) [inline plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.scss]: remove this.

Comment (jonnyalexbh) [reply plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.scss]: 👍

File: plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.scss (hunk 1 above)
Comment (jonnyalexbh) [reply plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.scss]: ready 🔢

**Review by betoobernal — APPROVED** [review]

**Pull request merged**